tests/unit/FirebaseArray.spec.js

```diff
@@ -40,13 +40,20 @@ describe('$firebaseArray', function () {
       $utils = $firebaseUtils;
       $rootScope = _$rootScope_;
       $q = _$q_;
-      tick = function (cb) {
+
+      firebase.database.enableLogging(function () {tick()});
+      tick = function () {
         setTimeout(function() {
           $q.defer();
           $rootScope.$digest();
-          cb && cb();
-        }, 1000)
+          try {
```

Comment:
Agreed - I'm not sure we call `flushAll` anywhere anymore, but I'll check, I'll migrate one way or the other.